.github/workflows/build-images-and-charts.yaml

```
@@ -0,0 +1,249 @@
+name: "Build Images and Charts"
+description: |
```

actionlint (1.7.9)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)